diesel/src/expression/functions/aggregate_folding.rs

```diff
@@ -72,23 +72,48 @@ fold_function!(
     avg,
     Avg,
     "AVG",
-    "Represents a SQL `AVG` function. This function can only take types which are
+    r#"Represents a SQL `AVG` function. This function can only take types which are
```

Comment:
What does the `r#` do here or is it a typo?

Reply:
This is a [raw string literal](https://doc.rust-lang.org/reference/tokens.html#raw-string-literals). You can use it when you have lots of quotes in a string and don't want to escape them. In addition to the usual quotes you add the same number of `#`s before and after the quotes.